timber-lint/src/main/java/timber/lint/WrongTimberUsageDetector.java

```diff
@@ -545,12 +546,20 @@ private boolean isLoggingExceptionMessage(JavaContext context, UExpression arg)
       return false;
     }
 
+    if (arg instanceof KotlinUQualifiedReferenceExpression) {
```

Comment:
How about keeping it as an interface check and identify that this is Kotlin like so:
```
    PsiElement psi = argExpression.getPsi();
    if (psi != null && LintUtils.isKotlin(psi.getLanguage()))
```

Reply:
I'd still need to cast it to access the selector and receiver. Maybe add this check when we need to handle more than just `KotlinUQualifiedReferenceExpression` and `UCallExpression`?

Reply:
Move the Kotlin language check after the following line which casts to argExpression.

Reply:
updated